apps/video/index.html

```diff
@@ -151,18 +172,20 @@ <h1 id="video-title"></h1>
     <!-- This element is the template of each thumbnail item -->
     <div id="thumbnail-group-header" class="hidden">
       <!--
-        <li>
-          <div class="thumbnail-group-header"></div>
-          <ul class="thumbnail-group-container"></ul>
+        <li role="presentation">
+          <div class="thumbnail-group-header" role="heading"
+               aria-level="1"></div>
+          <ul class="thumbnail-group-container" role="listbox"
```

Comment:
So here the header is for the group and it makes sense to expose it semantically as such. The thumbnail is represented as an option (actionable list item within a listbox). Our screen reader utters options in one sentence and treats them as atomic elements so we opt out for being terse there.